### PR TITLE
update BaiduProvider API

### PR DESCRIPTION
**Pull request by videni**

### CODE REVIEW

File: src/Mremi/UrlShortener/Provider/Baidu/BaiduProvider.php [hunk 1]
```diff
@@ -74,18 +74,18 @@ public function expand(LinkInterface $link)
...
+                    'shortUrl' => $link->getShortUrl(),
                 ),
             ),
             $this->options
         ));
 
         $response = $this->validate($response->getBody()->getContents(), true);
 
-        $link->setLongUrl($response->longurl);
+        $link->setLongUrl($response->ShortUrl);
```
Comment (mremi) [inline src/Mremi/UrlShortener/Provider/Baidu/BaiduProvider.php]: Looks weird

**Review by mremi — COMMENTED** [review]
Hi @videni , the CI is broken, can you have a look please?

**Comment by mremi** [comment]
@zacksleo hi, can you have a look to this PR please, any feedback? Thx

**Comment by zacksleo** [comment]
ok, I will check later.

**Comment by videni** [comment]
I  fixed the test, it should be ok now

**Comment by zacksleo** [comment]
@videni What is the difference between these two apis except the params? Why we should use this one?

File: src/Mremi/UrlShortener/Provider/Baidu/BaiduProvider.php (hunk 1 above)
Comment (zacksleo) [reply src/Mremi/UrlShortener/Provider/Baidu/BaiduProvider.php]: @mremi According the new [office api docs](https://dwz.cn/#/apidoc?_k=mtq3zn), every thing sames to be ok with the new PR. I have tested the api by the  cli, and it works.

image

@videni  Additional, can you update this line to ` 'base_uri' => 'https://dwz.cn',` ?

https://github.com/videni/UrlShortener/blob/c009b5110db45a0d1ae6d3510403b48b012c290f/src/Mremi/UrlShortener/Provider/Baidu/BaiduProvider.php#L102

**Comment by videni** [comment]
@zacksleo , their API changed, can you check their document please [百度短链](https://dwz.cn/#/apidoc?_k=o3zax0)?

**Comment by zacksleo** [comment]
Yeah, I have checked. Well done. Please see https://github.com/mremi/UrlShortener/pull/16#discussion_r224978911

**Comment by videni** [comment]
@zacksleo , fixed

**Comment by mremi** [comment]
Thx @videni and @zacksleo for the review 👍

**Pull request merged**